conda_libmamba_solver/solver.py

```diff
@@ -314,21 +315,42 @@ def _solving_loop(
 
     def _print_info(self):
         if not context.json and not context.quiet:
-            print(
-                dedent(
-                    f"""
-                    ----       USING EXPERIMENTAL LIBMAMBA INTEGRATIONS       ----
-                        This is a highly experimental product. If something is
-                        not working as expected, please submit an issue at
-                        https://github.com/conda/conda and attach the log file
-                        found in the following path. Thank you!
-
-                        {context._logfile_path}
+            msg = dedent(
+                f"""
+                ----       USING EXPERIMENTAL LIBMAMBA INTEGRATIONS       ----
+                    This is a highly experimental product. If something is
+                    not working as expected, please submit an issue at
+                    https://github.com/conda/conda and attach the log file
+                    found in the following path. Thank you!
+
+                    {context._logfile_path}
+
+                ---------------------------------------------------------------
+                """
+            ).rstrip()
+            if context.experimental_solver_confirmation:
+                risk = dedent(
+                    """
+                    There are risks using this experimental solver since it
+                    does not always produce identical results as that of the
+                    classical solver. As with any experimental/beta product it
+                    is not recommended to use this in production/critical
+                    environments.
 
-                    ---------------------------------------------------------------
+                    Do you wish to proceed
                     """
-                )
-            )
+                ).rstrip()
+                confirm_yn(f"{msg}\n{risk}", dry_run=False)
+
+                # update ~/.condarc to no longer prompt
+                with open(user_rc_path, "r+") as fh:
+                    condarc = yaml_round_trip_load(fh.read())
+                    condarc["experimental_solver_confirmation"] = False
+                    fh.seek(0)
+                    fh.write(yaml_round_trip_dump(condarc))
```

Comment:
Should we wrap this in a try/except block with `OSError` to handle any accidental permission issues?